demo.py

```diff
@@ -1,252 +1,56 @@
-import sys
-sys.path.append('submodules')
-
-from kivy.uix.widget import Widget
-from kivy.lang import Builder
-from kivy.uix.boxlayout import BoxLayout
-
-from flat_kivy.flatapp import FlatApp
-from flat_kivy.uix.flattextinput import FlatTextInput
-
-Builder.load_string('''
-#:import NavigationDrawer kivy_garden.navigationdrawer.NavigationDrawer
```

Comment:
this should really be kivy.garden.navigationdrawer.NavigationDrawer

Are you using a non-standard setup for garden?

Reply:
Right now yes, i am doing things in a non-standard way. Since the launchers I am using are not rebuilt w/ buildozer or any of the ios toolchain then anything that I want from the Kivy garden I have to put in manually somewhere rather than a traditional location (e.g. ending up in .../site-packages/kivy/garden or in my local ~/.kivy/garden.

Would you have a suggested way to handle this given that I want to use garden files w/o being able to install them the 'traditional' way?

Reply:
Ok I think given the situation it is best to do as you are doing.

If you want to get a little more robust:

In python you could wrap the imports in a try/except  and have it work both in your non-standard environment and a standard one.
```
try:
    from kivy.garden.navigationdrawer import NavigationDrawer
except:
    from kivy_garden.navigationdrawer import NavigationDrawer

```

but I don't think you can do the same thing in kv. You could emulate that by actually having a local project that does the above python snippet and then you import the symbol from that.

Reply:
If I create .kivy/garden in the root directory of the app and then move navigationdrawer to .kivy/garden/garden.navigationdrawer then I can use the standard import, and I guess this is where the buildozer process is putting garden files.

However this keeps the garden files local to the app (when running on Android) and what I guess I want is that somehow it is part of the module rather than the app.  So for garden files that should be common to anyone using flatkivyextensions does not have to worry about including the files in their app.

I would really like to make it so that the garden files we need are in standard locations so I will keep thinking about this....

Reply:
What are you hoping to gain by arranging things in this way? I want to better understand your thinking here.

Reply:
After some more thought I realized that these weird issues are all related to how our developers need to have the application, packages, and garden files prepared to use our launchers and nothing with the project we are working on here.

So I will move forward as though garden files are handled in the normal way and our packaging process will account for any massaging that needs to be done.

Sorry to have made this so confusing.